### PR TITLE
Add convenience methods to training container

### PR DESCRIPTION
**Pull request by pmrv**
As suggested by @JNmpi.

### CODE REVIEW

File: pyiron_contrib/atomistic/atomistics/job/trainingcontainer.py [hunk 1]
```diff
@@ -50,6 +52,18 @@ def __init__(self, project, job_name):
...
+    def include_job(self, job, iteration_step=-1):
+        """
+        Add structure, energy and forces from job.
+
+        Args:
+            job (:class:`.AtomisticGenericJob`): job to take structure from
+            iteration_step (int, optional): if job has multiple steps, this selects which to add
+        """
+        self.include_structure(job.get_structure(iteration_step=iteration_step),
+                               energy=job.output.energy_pot[-1], forces=job.output.forces[-1],
```
Comment (jan-janssen) [inline pyiron_contrib/atomistic/atomistics/job/trainingcontainer.py:64]: Should not this be iteration_step ? Otherwise it is inconsistent

Comment (pmrv) [reply pyiron_contrib/atomistic/atomistics/job/trainingcontainer.py]: Yes, I missed that.

Comment (pmrv) [reply pyiron_contrib/atomistic/atomistics/job/trainingcontainer.py]: I don't think I can add an easy test for this, though. :/

**Review by jan-janssen — APPROVED** [review]
Looks good to me

**Comment by jan-janssen** [comment]
@pmrv From my perspective this is ready to be merged.

**Pull request merged**